fawltydeps/packages.py

```diff
@@ -131,10 +136,18 @@ def packages(self) -> Dict[str, Package]:
         This enumerates the available packages  _once_, and caches the result for
         the remainder of this object's life in _packages.
         """
+        user_mapping: TomlData = {}
+
+        if self.mapping_path is not None:
+            logger.debug(f"Loading user-defined mapping from {self.mapping_path}")
+            with open(self.mapping_path, "rb") as mapping_file:
+                toml_data = tomllib.load(mapping_file)
+                user_mapping = {**user_mapping, **toml_data}
 
-        logger.debug(f"Loading user-defined mapping from {self.mapping_path}")
-        with open(self.mapping_path, "rb") as mapping_file:
-            user_mapping = tomllib.load(mapping_file)
+        if self.custom_mapping is not None:
+            logger.debug("Loading user-defined mapping from custom mapping.")
+            # Custom mapping overrides mapping from file
```

Comment:
You mean toml config custom mapping overrides custom mapping from file?

Reply:
Changed according to the comment below.